lib/injectGithubFacade.js

```diff
@@ -93,62 +101,142 @@ module.exports = ({ octokit, owner, repo, fs }) => ({
       repo,
       workflow_id: workflowId,
       ref,
-      inputs,
+      inputs
     });
   },
   async findIssueNumberByIssueTitle(title) {
-    const {
-      data: { items },
-    } = await octokit.search.issuesAndPullRequests({
-      q: `repo:${owner}/${repo} is:issue is:open in:title ${title}`,
-    });
-    const issue = items.find((i) => i.title === title);
-    if (issue) {
-      return issue.number;
-    }
-    throw new Error(`Could not find issue with title ${title}`);
+    const issue = getOneMatching(
+      octokit.search.issuesAndPullRequests,
+      { q: `repo:${owner}/${repo} is:issue is:open in:title ${title}` },
+      i => i.title === title
+    );
+    assert(issue, `Could not find issue with title ${title}.`);
+    return issue.number;
   },
   async createIssueComment(issueNumber, body) {
     await octokit.issues.createComment({
       owner,
       repo,
       issue_number: issueNumber,
-      body,
+      body
     });
   },
   async closeIssue(issueNumber) {
     await octokit.issues.update({
       owner,
       repo,
       issue_number: issueNumber,
-      state: "closed",
+      state: "closed"
     });
   },
   async createRelease({ tagName, name, body, prerelease }) {
     const {
-      data: { upload_url: uploadUrl },
+      data: { upload_url: uploadUrl }
     } = await octokit.repos.createRelease({
       owner,
       repo,
       tag_name: tagName,
       name,
       body,
-      prerelease,
+      prerelease
     });
     return uploadUrl;
   },
   async uploadReleaseAsset(url, filename) {
     const headers = {
       "content-type": mime.lookup(filename) || "application/octet-stream",
-      "content-length": (await fs.lstat(filename)).size,
+      "content-length": (await fs.lstat(filename)).size
     };
     const data = await fs.readFile(filename);
 
     await octokit.repos.uploadReleaseAsset({
       url,
       headers,
       name: path.basename(filename),
-      data,
+      data
     });
   },
+  /*
+  async getPullRequestsSinceLastRelease(ref) {
+    const commit = getOneMatching(
+      octokit.repos.listCommits,
+      { owner, repo, sha: ref },
+      c => {
+
+      }
+    )
+  },
+  async getTagSha(tagName) {
+    const tag = getOneMatching(
+      octokit.repos.listTags,
+      { owner, repo },
+      t => t.name === tagName
+    );
+    assert(tag, `Could not find tag matching: ${tagName}`);
+    return tag.sha;
+  } */
```

Comment:
Did you want to remove these?

Reply:
I wrote these before I found the power of the graph api. I didn't realize I left this in. I will remove.